config/install/user.role.state_admin.yml

```diff
@@ -9,25 +9,33 @@ dependencies:
   module:
     - config_translation
     - content_translation
+    - eck
     - eck_bundle_permissions
+    - form_mode_control
     - locale
     - rte_mis_core
     - rte_mis_school
     - taxonomy
+    - toolbar
     - workflow
 id: state_admin
 label: 'State Admin'
 weight: 3
 is_admin: null
 permissions:
   - 'access taxonomy overview'
+  - 'access toolbar'
+  - 'access user profiles'
+  - access_all_form_modes
+  - 'administer permissions'
```

Comment:
@kunal-sin Is it ok to give state admin, access to the permission page?

Reply:
@RohanSingh-inno Please confirm what all admin pages are visible to state admin.
I think we might have to remove this.

Reply:
@RohanSingh-inno Similar is added for district admin as well

Reply:
@jaswinsingh  State Admin should have permission to modify roles of existing users or create new user with user roles below its heirarchy level.

Reply:
@RohanSingh-inno If this admin permission is exposing some other features as well then let's remove this permission and create a new permission for this.

Reply:
> @RohanSingh-inno Similar is added for district admin as well

Yes because the district admin should have similar permission as the state admin for this case.